packages/components/src/FormField/FormField.css

```diff
@@ -85,11 +92,17 @@
   --field--padding-top: calc(var(--space-large) + var(--space-smaller));
 }
 
-.wrapper.textarea {
+.wrapper.text {
   --field--placeholder-offset: 0;
   --field--placeholder-transform: 0;
-  height: auto;
+  /* flex-direction: column; */
```

Comment:
Minor thing but I notice some of the CSS is commented out. It is probably worth removing these unused styles

Reply:
good catch. yeah I went through many iterations on the CSS and it kinda shows. I'll clean those up!